userguide/image_classifier/advanced-usage.md

```diff
@@ -20,7 +20,12 @@ annotated_data = tc.image_classifier.annotate(data)
 
 ```
 
-If you forget to assign the output of your annotation to a variable, 
+This utility will **only collect valid images**. All undefined values
```

Comment:
I'm not sure what "only collect valid images" means.

How about "only present rows from the SFrame that have non-missing images"? And I don't think I would make it bold... as a user, I would not think it's too surprising or needs to be taken note of, just good to know.

Reply:
Oh, sure.